.github/workflows/helm.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         uses: helm/chart-testing-action@v2.2.0
 
       - name: test token connectors chart templating
-        run: helm template charts/firefly --set "erc1155.enabled=true"
+        run: helm template charts/firefly --set "erc20.enabled=true"
```

Comment:
We'll want to set both here so we test templating for both connectors while we wait for ethconnect support.